d2renderers/d2svg/appendix/testdata/diagram_wider_than_tooltip/sketch.exp.svg

```diff
@@ -3,7 +3,7 @@
 id="d2-svg"
 style="background: white;"
 xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
-width="1963" height="2341" viewBox="-177 -49 1963 2341"><style type="text/css">
+width="1963" height="2421" viewBox="-177 -49 1963 2421"><style type="text/css">
```

Comment:
this changed in a bad way. i think order got messed up?

Reply:
Like order of the numbers?

Reply:
sry i don't mean to comment on the specific svg lines, just the pictures. 

by order, i mean order of sequence diagram edges in the svg

Reply:
Can confirm I see that too now.

Reply:
ohhh fuck we don't have global edge ordering anymore.

Reply:
ah, i guess the only place it would matter is for sequence diagrams.

Reply:
I guess I can write something to order the edges based on their first ref.

Reply:
Added but now the dotted lines are gone. See slack.